src/components/Notifications/TimeConfig.tsx

```diff
@@ -21,7 +21,7 @@ const dropDownPaddingClassName = style({
 
 export const TimeConfigComponent: React.FunctionComponent = () => {
 
-    const [ radioSelect, setRadioSelect ] = React.useState(true);
+    const [ radioSelect, setRadioSelect ] = React.useState(false);
```

Comment:
If I understand correctly, this one prevents selecting either of the radio elements until `timePref` has finished loading. I think we should do as follows:

Reply:
```suggestion
```

Reply:
`setRadioSelect` is used on at least 2 other places in this file and we should remove that too.